### PR TITLE
Added GetX package for state management

### PR DESCRIPTION
**Pull request by stefandevo**
As requested, @filiph @sfshaza2

### CODE REVIEW

**Comment by stefandevo** [comment]
@googlebot I signed it!

File: src/docs/development/data-and-backend/state-mgmt/options.md [hunk 1]
```diff
@@ -94,6 +94,14 @@ A popular library based on observables and reactions.
 * [Getting started with MobX.dart][]
 * [Flutter: State Management with Mobx][], a video by Paul Halliday
 
+## GetX
+
+GetX is an extra-light and powerful solution for Flutter. It combines high performance state management, intelligent dependency injection, and route management in a quick and practical way.
```
Comment (filiph) [inline src/docs/development/data-and-backend/state-mgmt/options.md:99]: I'm not disagreeing with any of the statements here, but this reads too much like a marketing paragraph. I would like this pared down to a short sentence with no superlatives, and with only one feature mentioned — something that differentiates it from other approaches on this page.

See other sections for inspiration.

Redux:

> A state container approach familiar to many web developers.

setState:

> The low-level approach to use for widget-specific, ephemeral state.

BLoC / Rx:

> A family of stream/observable based patterns.

**Comment by filiph** [comment]
Fixes https://github.com/flutter/website/issues/4478.

File: src/docs/development/data-and-backend/state-mgmt/options.md (hunk 1 above)
Comment (stefandevo) [reply src/docs/development/data-and-backend/state-mgmt/options.md]: Agree (was a copy paste from the readme). Changed it to 1 sentence. Is this oK?

**Comment by sfshaza2** [comment]
I think @filiph is off today, but I will wait for his LGTM.

**Pull request merged**